.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,17 @@
-* [ ] Have you followed the guidelines in our
-      [Contributing document](https://dvc.org/doc/user-guide/contributing/core)?
+* [ ] ❗ Have you followed the guidelines in the
+      [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core)
+      list?
 
-* [ ] Does your PR affect documented changes or does it add new functionality
-      that should be documented? If yes, have you created a PR for
-      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
-      least opened an issue for it? If so, please add a link to it.
+* [ ] 📖 This PR does _not_ require [documentation](https://dvc.org/doc) updates
+      or modifications. If it _does_, have you created a PR for
+      [dvc.org](https://github.com/iterative/dvc.org) or at least opened an
+      issue for it? If so, please add a link to it and after that check this
+      box.
```

Comment:
I get the idea but checking a _not_ item may be confusing, how about:

If this PR require [documentation](https://dvc.org/doc) updates, have you created a separate PR in [dvc.org](https://github.com/iterative/dvc.org), or at least opened an issue there about it? **Please link to either one in the description of this PR.**

Reply:
that's was the whole point of doing this PR :) It was not clear if one should check this box if documentation updates are not required. If one does not check it, PR looks incomplete and reviewers should think why is it not checked.

Reply:
there should be a clear intention to click this and be done with it if doc modification is not required - this is def majority of cases

Reply:
I thought the main point was to "mention recommendatory nature of CC and DS" 😋

> With the current (positive) template if one does not check it, PR looks incomplete

Good point. Maybe just be even more direct (and incorporate some of my wording) like:

Check this box if this PR **does not** require documentation updates, or if it does **and** you have created a **separate PR** in dvc.org with such updates (or at least opened an issue about it in that repo). Please link to your PR (or issue) in the documentation repository below.

Reply:
> I thought the main point was to "mention recommendatory nature of CC and DS" 

yeah, kinda ... hidden agenda :)

Reply:
done! took your wording, thanks!